guides/common/modules/proc_installing-capsule-server-packages.adoc

```diff
@@ -28,4 +37,27 @@ ifndef::satellite[]
 ----
 # {package-install} foreman-proxy-content
 ----
+
+ifdef::foreman-el,katello[]
+== [[centos-7-package]]CentOS 7
+
+:package-manager: yum
+include::proc_installing-proxy-packages-el.adoc[]
+
+endif::[]
+
+ifdef::foreman-el,katello[]
+== [[centos-8-package]]CentOS 8
+
+:package-manager: dnf
+include::proc_installing-proxy-packages-el.adoc[]
+
+endif::[]
+
+ifdef::foreman-el,katello,satellite[]
+== [[rhel-7-package]]Red Hat Enterprise Linux 7
```

Comment:
For non-Satellite builds, can we combine it into RHEL 7 / CentOS 7 or are they really different?